source/core/ut_metadata.pkb

```diff
@@ -134,12 +104,24 @@ create or replace package body ut_metadata as
   function get_package_spec_source(a_owner varchar2, a_object_name varchar2) return clob is
     l_txt_tab ut_varchar2_list;
     l_cur     sys_refcursor;
+    l_source  clob;
```

Comment:
I get your point.
We probably should not be using preprocessor for lookup of package body source line, that was called, though we could, but would need to fetch whole source just to get one line - not very optimal.